utils/swift_build_support/swift_build_support/shell.py

```diff
@@ -80,40 +88,74 @@ def call(command, stderr=None, env=None, dry_run=None):
             "': " + e.strerror)
 
 
+def capture(command, stderr=None, env=None, dry_run=None, print_command=True):
```

Comment:
Sorry, the default for `print_command` should be `False` **only for** capture.
I know, it's inconsistent with other methods,
but we rarely want to "trace" side-effect less query command.